File: src/main/java/cloud/elb/UpdateELBTree.java
package cloud.elb;

import se.sics.kompics.timer.ScheduleTimeout;
import se.sics.kompics.timer.Timeout;

/**
 * 
 * @author Amir Moulavi
 * @date 2011-05-14
 *
 */

public class UpdateELBTree extends Timeout {

	public UpdateELBTree(ScheduleTimeout request) {
		super(request);
	}

}

File: src/main/java/cloud/elb/UpdateELBTree.scala
package cloud.elb

import se.sics.kompics.timer.{Timeout, ScheduleTimeout}

/**
 * @author Amir Moulavi
 * @date 2011-07-12
 *
 */
class UpdateELBTree(request:ScheduleTimeout) extends Timeout(request)